### DISCUSSION TITLE
How can i get the token after payment?

### DISCUSSION THREAD

**Discussion by nc-kristema** [community/community#56294] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello together,
how can I get the token after payment? I have pay 2 $ for evcc.io (Homeassitant). But i have not get  a token.. No email with it. Where can i find it.

Best regards,

Markus

**Comment by SoundaryaKoutharapu** [comment]
I recommend visiting the official evcc.io or Home Assistant website to find their contact information or support channels. They will be in a better position to guide you on how to retrieve the token after payment.
